src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -247,7 +259,12 @@ export default class HaAutomationActionRow extends LitElement {
                       .path=${mdiRenameBox}
                     ></ha-svg-icon>
                   </mwc-list-item>
-                  <mwc-list-item graphic="icon" .disabled=${this.disabled}>
+                  <mwc-list-item
+                    graphic="icon"
+                    .disabled=${this.disabled}
+                    class=${classMap({ hidden: noReorderModeAvailable })}
+                    ?aria-hidden=${noReorderModeAvailable}
```

Comment:
1. `aria-hidden` is not a boolean attribute (it's a string that happens to take "true" or "false").  In fact, none of the ARIA attributes behave like booleans.
2. You don't need this anyway since you are styling as `display: none`.  User agents will not include anything with `display: none` or `visibility: hidden` in the accessibility tree.

Reply:
Thank you! I will update it!